### PR TITLE
Add php-flasher/flasher-symfony bundle recipe

### PR DESCRIPTION
**Pull request by yoeunes**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by yoeunes** [comment]
Hello @Nyholm can you review this PR please  ?

File: php-flasher/flasher-symfony/0.1/config/packages/flasher.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+                tailwindcss_bg:
+                    view: '@Flasher/tailwindcss_bg.html.twig'
+                    styles:
+                        - 'https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css'
+
+                bootstrap:
+                    view: '@Flasher/bootstrap.html.twig'
+                    styles:
+                        - 'https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.3/css/bootstrap.min.css'
```
Comment (Nyholm) [inline php-flasher/flasher-symfony/0.1/config/packages/flasher.yaml:20]: Could you tell me what this is?

Comment (yoeunes) [reply php-flasher/flasher-symfony/0.1/config/packages/flasher.yaml:20]: Hi @Nyholm thanks for your response.

the bundle contains a `Template` adapter by default that uses theses templates and their cdn files, which the user may want to add/edit, or remove if he wants to user local static files instead.

Comment (yoeunes) [reply php-flasher/flasher-symfony/0.1/config/packages/flasher.yaml:20]: I created this recipe for two reasons:
1 - I want this bundle to be registered automatically with other bundles that requires it like 'flasher/flasher-toastr-symfony'
2 - If I add this configuration in the `Configuration`  class : https://github.com/php-flasher/flasher-symfony/blob/main/DependencyInjection/Configuration.php

I lose the `template` key when ever I want to extends this bundle using : 
https://github.com/php-flasher/flasher-toastr-symfony/blob/main/DependencyInjection/FlasherToastrExtension.php

```php
$container->prependExtensionConfig('flasher', array(
      'adapters' => array(
          'toastr' => $config,
      ),
  ));
```

the desired config : 
```yaml
flasher:
    adapters:
        template: 
            # ......
        toastr:
           # .....
```
I don't know if you have any suggestion for this.
Thank you @Nyholm

**Pull request closed without merging**